moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js

```diff
@@ -240,9 +240,6 @@ export const ProjectsListViewQueryConf = {
   },
   // This object gets consumed into the GQLAbstract system, and here is the single, un-nested order_by directive. ✅
   order_by: { updated_at: "desc" },
-  where: {
```

Comment:
i was getting a weird syntax error that resulted in a wsod that @johnclary was eventually able to reproduce, i think yall already discussed it but apparently it has to do with setting this `where` to `null` instead of removing it :0

Reply:
thanks for catching this @roseeichelmann and for the fix! i was able to reproduce the issue and setting the ``where`` to ``null`` solved the problem! i've updated the pr